src/Models/ShallowWaterModels/shallow_water_advection_operators.jl

```diff
@@ -50,3 +50,10 @@ which will end up at the location `ccc`.
     1/Vᵃᵃᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_ψᵃᵃᶜ, solution.uh) + 
                              δyᵃᶜᵃ(i, j, k, grid, Ay_ψᵃᵃᶜ, solution.vh))
 end
+
+@inline function div_ucvc(i, j, k, grid, u, v, c)
```

Comment:
Would `div_Uc` be more consistent with `div_hUu` and `div_hUv` from earlier in this file?

Reply:
I agree, that is a better choice.  I suppose I was looking at the mass equation where we use `div_huhv`, which is not consistent.  I wonder if thta should be changed to `div_hU`?